### ISSUE TITLE
[BUG] Incorrect Number of Delay Files Written

### ISSUE THREAD

**Issue by cmarshak** [dbekaert/RAiDER#587]
Here are a small group of jobs (~329 / 90k) that failed in Raider.

The log is:
```
Total number of grid points (3D): 894672
=====================================
Extent of the weather model is (xmin, ymin, xmax, ymax):-83.79, 25.86, -79.01, 29.34
Extent of the input is (xmin, ymin, xmax, ymax): -82.70, 26.65, -79.65, 28.60
Processing slice 1 / 20: -500.0
INFO:RAiDER:Processing slice 1 / 20: -500.0
Processing slice 2 / 20: 0.0
INFO:RAiDER:Processing slice 2 / 20: 0.0
Processing slice 3 / 20: 500.0
INFO:RAiDER:Processing slice 3 / 20: 500.0
Processing slice 4 / 20: 1000.0
INFO:RAiDER:Processing slice 4 / 20: 1000.0
Processing slice 5 / 20: 1500.0
INFO:RAiDER:Processing slice 5 / 20: 1500.0
Processing slice 6 / 20: 2000.0
INFO:RAiDER:Processing slice 6 / 20: 2000.0
Processing slice 7 / 20: 2500.0
INFO:RAiDER:Processing slice 7 / 20: 2500.0
Processing slice 8 / 20: 3000.0
INFO:RAiDER:Processing slice 8 / 20: 3000.0
Processing slice 9 / 20: 3500.0
INFO:RAiDER:Processing slice 9 / 20: 3500.0
Processing slice 10 / 20: 4000.0
INFO:RAiDER:Processing slice 10 / 20: 4000.0
Processing slice 11 / 20: 4500.0
INFO:RAiDER:Processing slice 11 / 20: 4500.0
Processing slice 12 / 20: 5000.0
INFO:RAiDER:Processing slice 12 / 20: 5000.0
Processing slice 13 / 20: 5500.0
INFO:RAiDER:Processing slice 13 / 20: 5500.0
Processing slice 14 / 20: 6000.0
INFO:RAiDER:Processing slice 14 / 20: 6000.0
Processing slice 15 / 20: 6500.0
INFO:RAiDER:Processing slice 15 / 20: 6500.0
Processing slice 16 / 20: 7000.0
INFO:RAiDER:Processing slice 16 / 20: 7000.0
Processing slice 17 / 20: 7500.0
INFO:RAiDER:Processing slice 17 / 20: 7500.0
Processing slice 18 / 20: 8000.0
INFO:RAiDER:Processing slice 18 / 20: 8000.0
Processing slice 19 / 20: 8500.0
INFO:RAiDER:Processing slice 19 / 20: 8500.0
Processing slice 20 / 20: 9000.0
INFO:RAiDER:Processing slice 20 / 20: 9000.0
Successfully wrote delay cube to: /home/raider/HRRR_tropo_20171215T232847_ray.nc
INFO:RAiDER:
Successfully wrote delay cube to: /home/raider/HRRR_tropo_20171215T232847_ray.nc
Starting to run the weather model calculation
DEBUG:RAiDER:Starting to run the weather model calculation
Requested date,time: 20161126, 23:28
DEBUG:RAiDER:Requested date,time: 20161126, 23:28
Beginning weather model pre-processing
DEBUG:RAiDER:Beginning weather model pre-processing
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
ðŸ’” Did not find â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2016-Nov-26 23:00 UTC[92m F00[0m
[31;21mERROR: 
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`[0m
ERROR:RAiDER:
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`
[33;21mWARNING: [0m
WARNING:RAiDER:
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
ðŸ’” Did not find â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2016-Nov-27 00:00 UTC[92m F00[0m
[31;21mERROR: 
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`[0m
ERROR:RAiDER:
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`
[33;21mWARNING: [0m
WARNING:RAiDER:
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
Weather model HRRR is available from 2016-07-15 to Present
INFO:RAiDER:Weather model HRRR is available from 2016-07-15 to Present
ðŸ’” Did not find â”Š model=hrrr â”Š [3mproduct=nat[0m â”Š [38;2;41;130;13m2016-Nov-26 22:00 UTC[92m F00[0m
[31;21mERROR: 
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`[0m
ERROR:RAiDER:
No index file was found for None
Download the full file first (with `H.download()`).
You will need to remake the Herbie object (H = `Herbie()`)
or delete this cached property: `del H.index_as_dataframe()`
[33;21mWARNING: [0m
WARNING:RAiDER:
[31;21mERROR: No weather model data was successfully processed.[0m
ERROR:RAiDER:No weather model data was successfully processed.
Traceback (most recent call last):
  File "/opt/conda/envs/RAiDER/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/conda/envs/RAiDER/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/__main__.py", line 46, in <module>
    main()
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/__main__.py", line 42, in main
    process_entry_point.load()()
  File "/opt/conda/envs/RAiDER/lib/python3.10/site-packages/RAiDER/cli/raider.py", line 622, in calcDelaysGUNW
    assert len(cube_filenames) == 2, 'Incorrect number of delay files written.'
AssertionError: Incorrect number of delay files written.
```

Below are the inputs. Here is the GUNW for the first job linked below: https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/0f0a0a7b-7c30-4784-98c0-08f488c34a63/S1-GUNW-A-R-048-tops-20171215_20161126-232838-00083W_00027N-PP-d8ee-v3_0_0.nc

```
[{'job_id': '0f0a0a7b-7c30-4784-98c0-08f488c34a63',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7341,
   'granules': ['S1A_IW_SLC__1SDV_20171215T232812_20171215T232840_019720_02185F_1DE6',
    'S1A_IW_SLC__1SDV_20171215T232838_20171215T232905_019720_02185F_0EF7'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232808_20161126T232836_014120_016CA2_425D',
    'S1A_IW_SLC__1SDV_20161126T232834_20161126T232901_014120_016CA2_A775'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/0f0a0a7b-7c30-4784-98c0-08f488c34a63/0f0a0a7b-7c30-4784-98c0-08f488c34a63.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [8649.365, 399.566]},
 {'job_id': 'd18dd2e4-5990-455c-a7ba-f7c87ff2d757',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7349,
   'granules': ['S1A_IW_SLC__1SDV_20171215T233043_20171215T233110_019720_02185F_6A7E',
    'S1A_IW_SLC__1SDV_20171215T233108_20171215T233135_019720_02185F_3563'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T233039_20161126T233106_014120_016CA2_D9AD',
    'S1A_IW_SLC__1SDV_20161126T233104_20161126T233131_014120_016CA2_CF09'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/d18dd2e4-5990-455c-a7ba-f7c87ff2d757/d18dd2e4-5990-455c-a7ba-f7c87ff2d757.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [9359.326, 397.565]},
 {'job_id': '573f7049-c2b3-4b88-bfe2-022d9069e596',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7348,
   'granules': ['S1A_IW_SLC__1SDV_20171215T233017_20171215T233045_019720_02185F_D739',
    'S1A_IW_SLC__1SDV_20171215T233043_20171215T233110_019720_02185F_6A7E'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8',
    'S1A_IW_SLC__1SDV_20161126T233039_20161126T233106_014120_016CA2_D9AD'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/573f7049-c2b3-4b88-bfe2-022d9069e596/573f7049-c2b3-4b88-bfe2-022d9069e596.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [10214.2, 388.986]},
 {'job_id': '24117e9b-680a-4678-9c63-d0c0e57f4280',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7348,
   'granules': ['S1A_IW_SLC__1SDV_20171203T233018_20171203T233045_019545_0212EB_54DF',
    'S1A_IW_SLC__1SDV_20171203T233043_20171203T233110_019545_0212EB_4821'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8',
    'S1A_IW_SLC__1SDV_20161126T233039_20161126T233106_014120_016CA2_D9AD'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/24117e9b-680a-4678-9c63-d0c0e57f4280/24117e9b-680a-4678-9c63-d0c0e57f4280.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [10238.139, 385.829]},
 {'job_id': 'e2b41e83-53c1-4595-98f9-22a26f9ab35d',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7345,
   'granules': ['S1A_IW_SLC__1SDV_20171215T232928_20171215T232955_019720_02185F_421C',
    'S1A_IW_SLC__1SDV_20171215T232952_20171215T233019_019720_02185F_014C'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232923_20161126T232950_014120_016CA2_E7F1',
    'S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/e2b41e83-53c1-4595-98f9-22a26f9ab35d/e2b41e83-53c1-4595-98f9-22a26f9ab35d.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [9322.364, 387.862]},
 {'job_id': '648b11c4-7b7d-4880-8d15-905226aed7ab',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7345,
   'granules': ['S1A_IW_SLC__1SDV_20171203T232928_20171203T232955_019545_0212EB_AB31',
    'S1A_IW_SLC__1SDV_20171203T232953_20171203T233020_019545_0212EB_99DF'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232923_20161126T232950_014120_016CA2_E7F1',
    'S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/648b11c4-7b7d-4880-8d15-905226aed7ab/648b11c4-7b7d-4880-8d15-905226aed7ab.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [8904.252, 397.084]},
 {'job_id': '4f0e14b6-e47e-4fb6-91ae-8a8091a8c583',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7346,
   'granules': ['S1A_IW_SLC__1SDV_20171203T232953_20171203T233020_019545_0212EB_99DF',
    'S1A_IW_SLC__1SDV_20171203T233018_20171203T233045_019545_0212EB_54DF'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC',
    'S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/4f0e14b6-e47e-4fb6-91ae-8a8091a8c583/4f0e14b6-e47e-4fb6-91ae-8a8091a8c583.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [8769.394, 382.507]},
 {'job_id': '97a3951a-b402-4056-8b3f-8710996d9568',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7347,
   'granules': ['S1A_IW_SLC__1SDV_20171215T232952_20171215T233019_019720_02185F_014C',
    'S1A_IW_SLC__1SDV_20171215T233017_20171215T233045_019720_02185F_D739'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC',
    'S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/97a3951a-b402-4056-8b3f-8710996d9568/97a3951a-b402-4056-8b3f-8710996d9568.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [10150.097, 385.88]},
 {'job_id': 'b82b7c80-3ec4-42bb-b445-fd42c9781f31',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7340,
   'granules': ['S1A_IW_SLC__1SDV_20171215T232747_20171215T232814_019720_02185F_0412',
    'S1A_IW_SLC__1SDV_20171215T232812_20171215T232840_019720_02185F_1DE6'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232740_20161126T232810_014120_016CA2_DA39',
    'S1A_IW_SLC__1SDV_20161126T232808_20161126T232836_014120_016CA2_425D'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/b82b7c80-3ec4-42bb-b445-fd42c9781f31/b82b7c80-3ec4-42bb-b445-fd42c9781f31.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [9245.797, 392.316]},
 {'job_id': '392fe9fe-01f2-48d4-94bc-1967f3c78ecf',
  'job_type': 'INSAR_ISCE',
  'request_time': '2023-08-25T18:42:59+00:00',
  'status_code': 'FAILED',
  'user_id': 'access_cloud_based_insar',
  'name': 'track48-n3',
  'job_parameters': {'compute_solid_earth_tide': True,
   'estimate_ionosphere_delay': True,
   'frame_id': 7346,
   'granules': ['S1A_IW_SLC__1SDV_20171215T232952_20171215T233019_019720_02185F_014C',
    'S1A_IW_SLC__1SDV_20171215T233017_20171215T233045_019720_02185F_D739'],
   'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC',
    'S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8'],
   'weather_model': 'HRRR'},
  'logs': ['https://hyp3-tibet-jpl-contentbucket-81rn23hp7ppf.s3.us-west-2.amazonaws.com/392fe9fe-01f2-48d4-94bc-1967f3c78ecf/392fe9fe-01f2-48d4-94bc-1967f3c78ecf.log'],
  'expiration_time': '2023-10-28T00:00:00+00:00',
  'processing_times': [9722.873, 388.472]}]
```

**Comment by bbuzz31** [comment]
It looks like there are three dates causing all these failures: ['20161126', '20171203', '20171215']
I'm almost certain that they will be missing on the server, will try and confirm

**Comment by cmarshak** [comment]
@bbuzz31 - are the entire dates not available or just specific datetimes within those dates not available? It appears the amount of datetimes that are unavailable for HRRR is problematic. I am rather surprised. You mentioned it is partly because we are using a certain type of HRRR file with less availability?

**Comment by jlmaurer** [comment]
@cmarshak @bbuzz31 is this waiting for #585 to be merged in order to be resolved?

**Comment by cmarshak** [comment]
This has now been resolved with #593 and #597.

Here are the jobs taken directly from the issue ticket:

```
{'browse_images': ['https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/b0ab94e5-f573-44d5-8170-42e36a270ecc/S1-GUNW-A-R-048-tops-20171215_20161126-232838-00083W_00027N-PP-d8ee-v3_0_0.png'],
 'expiration_time': '2024-03-25T00:00:00+00:00',
 'files': [{'filename': 'S1-GUNW-A-R-048-tops-20171215_20161126-232838-00083W_00027N-PP-d8ee-v3_0_0.nc',
            's3': {'bucket': 'hyp3-a19-jpl-contentbucket-1wfnatpznlg8b',
                   'key': 'b0ab94e5-f573-44d5-8170-42e36a270ecc/S1-GUNW-A-R-048-tops-20171215_20161126-232838-00083W_00027N-PP-d8ee-v3_0_0.nc'},
            'size': 88430739,
            'url': 'https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/b0ab94e5-f573-44d5-8170-42e36a270ecc/S1-GUNW-A-R-048-tops-20171215_20161126-232838-00083W_00027N-PP-d8ee-v3_0_0.nc'}],
 'job_id': 'b0ab94e5-f573-44d5-8170-42e36a270ecc',
 'job_parameters': {'compute_solid_earth_tide': True,
                    'estimate_ionosphere_delay': True,
                    'frame_id': 7341,
                    'granules': ['S1A_IW_SLC__1SDV_20171215T232812_20171215T232840_019720_02185F_1DE6',
                                 'S1A_IW_SLC__1SDV_20171215T232838_20171215T232905_019720_02185F_0EF7'],
                    'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232808_20161126T232836_014120_016CA2_425D',
                                           'S1A_IW_SLC__1SDV_20161126T232834_20161126T232901_014120_016CA2_A775'],
                    'weather_model': 'HRRR'},
 'job_type': 'INSAR_ISCE_TEST',
 'logs': [],
 'name': 'issue_587!',
 'processing_times': [9202.845, 13.281],
 'request_time': '2023-09-26T20:06:28+00:00',
 'status_code': 'SUCCEEDED',
 'thumbnail_images': [],
 'user_id': 'cmarshak'}
{'browse_images': ['https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/331b1aff-2941-435b-8d0a-73cd5ffef0dc/S1-GUNW-A-R-048-tops-20171203_20161126-232954-00084W_00031N-PP-ad06-v3_0_0.png'],
 'expiration_time': '2024-03-25T00:00:00+00:00',
 'files': [{'filename': 'S1-GUNW-A-R-048-tops-20171203_20161126-232954-00084W_00031N-PP-ad06-v3_0_0.nc',
            's3': {'bucket': 'hyp3-a19-jpl-contentbucket-1wfnatpznlg8b',
                   'key': '331b1aff-2941-435b-8d0a-73cd5ffef0dc/S1-GUNW-A-R-048-tops-20171203_20161126-232954-00084W_00031N-PP-ad06-v3_0_0.nc'},
            'size': 91540832,
            'url': 'https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/331b1aff-2941-435b-8d0a-73cd5ffef0dc/S1-GUNW-A-R-048-tops-20171203_20161126-232954-00084W_00031N-PP-ad06-v3_0_0.nc'}],
 'job_id': '331b1aff-2941-435b-8d0a-73cd5ffef0dc',
 'job_parameters': {'compute_solid_earth_tide': True,
                    'estimate_ionosphere_delay': True,
                    'frame_id': 7345,
                    'granules': ['S1A_IW_SLC__1SDV_20171203T232928_20171203T232955_019545_0212EB_AB31',
                                 'S1A_IW_SLC__1SDV_20171203T232953_20171203T233020_019545_0212EB_99DF'],
                    'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232923_20161126T232950_014120_016CA2_E7F1',
                                           'S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC'],
                    'weather_model': 'HRRR'},
 'job_type': 'INSAR_ISCE_TEST',
 'logs': [],
 'name': 'issue_587!',
 'processing_times': [9412.404, 13.444],
 'request_time': '2023-09-26T20:06:28+00:00',
 'status_code': 'SUCCEEDED',
 'thumbnail_images': [],
 'user_id': 'cmarshak'}
{'browse_images': ['https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/f363ba04-9dec-403d-8cbb-017752b39065/S1-GUNW-A-R-048-tops-20171215_20161126-232813-00082W_00026N-PP-f12f-v3_0_0.png'],
 'expiration_time': '2024-03-25T00:00:00+00:00',
 'files': [{'filename': 'S1-GUNW-A-R-048-tops-20171215_20161126-232813-00082W_00026N-PP-f12f-v3_0_0.nc',
            's3': {'bucket': 'hyp3-a19-jpl-contentbucket-1wfnatpznlg8b',
                   'key': 'f363ba04-9dec-403d-8cbb-017752b39065/S1-GUNW-A-R-048-tops-20171215_20161126-232813-00082W_00026N-PP-f12f-v3_0_0.nc'},
            'size': 87397293,
            'url': 'https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/f363ba04-9dec-403d-8cbb-017752b39065/S1-GUNW-A-R-048-tops-20171215_20161126-232813-00082W_00026N-PP-f12f-v3_0_0.nc'}],
 'job_id': 'f363ba04-9dec-403d-8cbb-017752b39065',
 'job_parameters': {'compute_solid_earth_tide': True,
                    'estimate_ionosphere_delay': True,
                    'frame_id': 7340,
                    'granules': ['S1A_IW_SLC__1SDV_20171215T232747_20171215T232814_019720_02185F_0412',
                                 'S1A_IW_SLC__1SDV_20171215T232812_20171215T232840_019720_02185F_1DE6'],
                    'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232740_20161126T232810_014120_016CA2_DA39',
                                           'S1A_IW_SLC__1SDV_20161126T232808_20161126T232836_014120_016CA2_425D'],
                    'weather_model': 'HRRR'},
 'job_type': 'INSAR_ISCE_TEST',
 'logs': [],
 'name': 'issue_587!',
 'processing_times': [9076.019, 13.923],
 'request_time': '2023-09-26T20:06:28+00:00',
 'status_code': 'SUCCEEDED',
 'thumbnail_images': [],
 'user_id': 'cmarshak'}
{'browse_images': ['https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/5dc5cbde-c9d3-460a-999f-6293723868a7/S1-GUNW-A-R-048-tops-20171203_20161126-233019-00084W_00033N-PP-dce8-v3_0_0.png'],
 'expiration_time': '2024-03-25T00:00:00+00:00',
 'files': [{'filename': 'S1-GUNW-A-R-048-tops-20171203_20161126-233019-00084W_00033N-PP-dce8-v3_0_0.nc',
            's3': {'bucket': 'hyp3-a19-jpl-contentbucket-1wfnatpznlg8b',
                   'key': '5dc5cbde-c9d3-460a-999f-6293723868a7/S1-GUNW-A-R-048-tops-20171203_20161126-233019-00084W_00033N-PP-dce8-v3_0_0.nc'},
            'size': 93062346,
            'url': 'https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/5dc5cbde-c9d3-460a-999f-6293723868a7/S1-GUNW-A-R-048-tops-20171203_20161126-233019-00084W_00033N-PP-dce8-v3_0_0.nc'}],
 'job_id': '5dc5cbde-c9d3-460a-999f-6293723868a7',
 'job_parameters': {'compute_solid_earth_tide': True,
                    'estimate_ionosphere_delay': True,
                    'frame_id': 7346,
                    'granules': ['S1A_IW_SLC__1SDV_20171203T232953_20171203T233020_019545_0212EB_99DF',
                                 'S1A_IW_SLC__1SDV_20171203T233018_20171203T233045_019545_0212EB_54DF'],
                    'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC',
                                           'S1A_IW_SLC__1SDV_20161126T233013_20161126T233041_014120_016CA2_B4C8'],
                    'weather_model': 'HRRR'},
 'job_type': 'INSAR_ISCE_TEST',
 'logs': [],
 'name': 'issue_587!',
 'processing_times': [9483.527, 12.708],
 'request_time': '2023-09-26T20:06:28+00:00',
 'status_code': 'SUCCEEDED',
 'thumbnail_images': [],
 'user_id': 'cmarshak'}
{'browse_images': ['https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/8c5c8542-dfc2-4f47-8a46-2c74f8297d61/S1-GUNW-A-R-048-tops-20171215_20161126-232953-00084W_00031N-PP-012e-v3_0_0.png'],
 'expiration_time': '2024-03-25T00:00:00+00:00',
 'files': [{'filename': 'S1-GUNW-A-R-048-tops-20171215_20161126-232953-00084W_00031N-PP-012e-v3_0_0.nc',
            's3': {'bucket': 'hyp3-a19-jpl-contentbucket-1wfnatpznlg8b',
                   'key': '8c5c8542-dfc2-4f47-8a46-2c74f8297d61/S1-GUNW-A-R-048-tops-20171215_20161126-232953-00084W_00031N-PP-012e-v3_0_0.nc'},
            'size': 91540924,
            'url': 'https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/8c5c8542-dfc2-4f47-8a46-2c74f8297d61/S1-GUNW-A-R-048-tops-20171215_20161126-232953-00084W_00031N-PP-012e-v3_0_0.nc'}],
 'job_id': '8c5c8542-dfc2-4f47-8a46-2c74f8297d61',
 'job_parameters': {'compute_solid_earth_tide': True,
                    'estimate_ionosphere_delay': True,
                    'frame_id': 7345,
                    'granules': ['S1A_IW_SLC__1SDV_20171215T232928_20171215T232955_019720_02185F_421C',
                                 'S1A_IW_SLC__1SDV_20171215T232952_20171215T233019_019720_02185F_014C'],
                    'secondary_granules': ['S1A_IW_SLC__1SDV_20161126T232923_20161126T232950_014120_016CA2_E7F1',
                                           'S1A_IW_SLC__1SDV_20161126T232948_20161126T233015_014120_016CA2_76EC'],
                    'weather_model': 'HRRR'},
 'job_type': 'INSAR_ISCE_TEST',
 'logs': [],
 'name': 'issue_587!',
 'processing_times': [9513.125, 13.301],
 'request_time': '2023-09-26T20:06:28+00:00',
 'status_code': 'SUCCEEDED',
 'thumbnail_images': [],
 'user_id': 'cmarshak'}
```
